src/lib/menu/menu-trigger.ts

```diff
@@ -70,6 +70,7 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
     if (!this._menuOpen) {
       this._createOverlay();
       this._overlayRef.attach(this._portal);
+      this._overlayRef.backdropClick().subscribe(() => this.closeMenu());
```

Comment:
There's an e2e that covers this already, but I can add a unit too if you think it will help coverage?